Solutions/Jamf Protect/Analytic Rules/JamfProtectAlerts.yaml

```diff
@@ -0,0 +1,81 @@
+id: 6098daa0-f05e-44d5-b5a0-913e63ba3179
+name: Jamf Protect - Alerts 
+description: |
+  'Creates an incident based on Jamf Protect Alert data in Microsoft Sentinel'
+severity: High
+status: Available
+requiredDataConnectors:
+  - connectorId: JamfProtect
+    dataTypes: 
+      - jamfprotect_CL
+suppressionDuration: PT5H
+suppressionEnabled: false
+tactics:
+techniques:
+alertRuleTemplateName:
+query: |
+  jamfprotect_CL
+  | where topicType_s == "alert"
+  | where input_eventType_s <> "GPUnifiedLogEvent"
+  | extend severity = case(input_match_severity_d == 0, "Informational", input_match_severity_d == 1, "Low", input_match_severity_d == 2, "Medium", input_match_severity_d == 3, "High", "True")
+  | extend Analytic_description =  parse_json(input_match_facts_s)[0].human
+  | extend Analytic_name =  parse_json(input_match_facts_s)[0].name
+  | extend AlertURL = strcat("https://", org_hd_s, ".jamfcloud.com/Alerts/", input_match_uuid_g)
+  | extend Input_Match = parse_json(input_match_actions_s)[0].name
+  | extend sha256hex = tostring(parse_json(input_related_binaries_s)[0].sha256hex)
+  | extend algorithm = "SHA256"
+  | extend Matched_Users = tostring(parse_json(input_related_users_s)[0].name)
+  | extend Host_IPs = tostring(parse_json(input_host_ips_s)[0])
+  | extend Tags = tostring(parse_json(input_match_facts_s)[0].tags)
+  | extend JamfPro = case(Input_Match contains "SmartGroup", "Workflow with Jamf Pro", Input_Match == "Prevented", "No workflow, Prevented by Protect", Input_Match == "", "No workflow", "No workflow")
+  | where isnotempty(severity)
```

Comment:
It would be better to filter the results first and then parse the rest of the fields.